neonvm/runner/main.go

```diff
@@ -1217,6 +1231,16 @@ func defaultNetwork(logger *zap.Logger, cidr string, ports []vmv1.Port) (mac.MAC
 		return nil, err
 	}
 
+	f, err := os.OpenFile("/etc/hosts", os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0644)
```

Comment:
It's not immediately clear what this section does. Let's add a brief comment?

Reply:
Fixed in 9c2169d044dc64d2921ae400688a79bc0bf1df25